src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -219,8 +218,14 @@
     {% set extraVars = extraVars|default({}) %}
 
     {% if extraVars.label is defined %}
+      {% set isRequired = form.vars.required|default(false) %}
+
+      {% if extraVars.required is defined %}
```

Comment:
Can you explain the usecase where this is needed ?

Reply:
"Change password" form type is shown as not required until you click on it, then it expands into a block with three required inputs. There was a problem with making those three inputs required while keeping the parent as not required, since in symfony forms if the parent form is not required, it's children will always be not required as well.